concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
No, if `post-deploy` depends on `avail tests` than during first run `avail-test` will fail and `post deploy` wont be run so health-check app will not be deployed.